{"name": "test_delete_evidence", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <test_case_details.TestCaseDetails object at 0x0162C750>\n\n    def test_delete_evidence(self):\n        driver = self.driver\n        self.login()\n        case_details = CaseDetails(driver)\n        home = HomePage(driver)\n        home.click_law_suit_page()\n        case_details.evidence_tab_entry()\n>       case_details.delete_evidence_attachment()\n\ntest_case_details.py:78: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.CaseDetailsPage.CaseDetails object at 0x0E7A0610>\n\n    def delete_evidence_attachment(self):\n        self.driver.find_element_by_xpath(self.delete_evidence_attachment_btn_xpath).click()\n        time.sleep(2)\n        # There should be a warning popup\n>       assert \"There should be a warning popup\" == \"There should be a warning\"\nE       AssertionError\n\n..\\Pages\\CaseDetailsPage.py:221: AssertionError"}, "start": 1568730444473, "stop": 1568730463509, "uuid": "7e72f7de-7741-4740-9056-ab69688c60ed", "historyId": "fda4845cb7aa8152e3b454e384844763", "testCaseId": "c8a0df3bb95de312caaaeaafdd5f535f", "fullName": "test_case_details.TestCaseDetails#test_delete_evidence", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_case_details"}, {"name": "subSuite", "value": "TestCaseDetails"}, {"name": "host", "value": "EVT01066NB"}, {"name": "thread", "value": "11132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_details"}]}